new/LIT6705CompGM.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6705CompGM" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">The Synaxarion-type compilation of ʾAbuna Gabra Māryām of Dabra Bizan</title>
+                <title xml:lang="en" xml:id="t2">Sǝnkǝssār of Dabra Bizan</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A compilatory work of hagiographical notices and biblical excerpts, arranged in commemorations according to the days of the year as the synaxarion. It contains
+                    commemorations that seem to be based upon those of <ref type="work" corresp="LIT4033Senkessar"/>, others that are very close
+                    to those not present in the first recension, but added in <ref type="work" corresp="LIT4035SenkessarDL"/>,
+                    and yet others that probably have different sources.
+                    The work is attested in <ref type="mss" corresp="DW03"/>, where <persName ref="PRS13864GabraM"/> is named as its compiler. 
```

Comment:
Isn't it worth saying that the work is attested in a ms that originates from Dabra Bizan, and therefore might represent a synaxarion of Dabra Bizan (to be somehow reflected in "Relations")? Big monasteries of Ethiopia may have their own versions of Synaxaria (cp. that of Gunda Gune).